src/main/java/net/dries007/tfc/TerraFirmaCraft.java

```diff
@@ -155,6 +159,15 @@ public void init(FMLInitializationEvent event)
             log.warn("You are not running an official build. Please do not use this and then report bugs or issues.");
         }
 
+        if (event.getSide() == Side.CLIENT)
```

Comment:
I'm not convinced this won't just crash on a dedicated server. That's the whole point of having a proxy to do client / server specific event handling

Reply:
Missed the lines below which already check for client side.